server/utils/passport/local-login.js

```diff
@@ -1,91 +1,56 @@
-import passport from 'passport';
-import config from '../../configs/config';
-import { MongoClient } from 'mongodb';
-import { verifyHash } from '../crypto/hash';
-import { getMongoURI } from '../mongoUtil';
-import defaultUserConfig from '../../configs/defaultUserConfig';
-import basePathConfig from '../../configs/basePathConfig';
+import passport from 'passport'
+import { collections } from '../mongoCollections'
+import { verifyHash } from '../crypto/hash'
+import defaultUserConfig from '../../configs/defaultUserConfig'
+import basePathConfig from '../../configs/basePathConfig'
 
-const basePath = basePathConfig || '';
+const basePath = basePathConfig || ''
 
 export default (req, res, next, user) => {
   //validate submitted password
   if (!verifyHash(req.body.password, user.password)) {
-    return res.redirect(`${basePath}/login?e=forbidden`);
+    return res.redirect(`${basePath}/login?e=forbidden`)
   }
   //passport local log-in serializer
   passport.serializeUser(function (user, done) {
-    done(null, user.uid);
-  });
+    return done(null, user.uid)
+  })
   passport.deserializeUser(function (user, done) {
-    done(null, user);
-  });
+    done(null, user)
```

Comment:
should we return here?

Reply:
https://github.com/AMP-SCZ/dpdash/pull/237#:~:text=%E2%80%A6-,7930718,-Merge%20state